.github/ISSUE_TEMPLATE/feature_request_template.md

```diff
@@ -0,0 +1,19 @@
+---
+name: Feature Request
+about: Use this template for feature and enhancement proposals.
+labels: 'enhancement'
+title: [FR]
+---
+Thank you for submitting an issue. Please refer to our [issue policy](https://www.github.com/mlflow/mlflow/blob/master/ISSUE_POLICY.md)
+for information on what types of issues we address.
+  
+Please do not delete this template unless you are sure your issue is outside its scope.
+
+### Background information
+- **MLflow version (run ``mlflow --version``)**:
+- **Python version**: 
+- **Am I willing to create a pull request if approved?**
+
+### Describe the proposal
+Describe the problem clearly here. What is the use case in mind?
```

Comment:
```suggestion

Provide a clear high-level description of the feature request in the following sections. Feature requests that are likely to be accepted:
* Are minimal in scope (note that it's always easier to add additional functionality later than remove functionality)
* Are extensible (e.g. if adding an integration with an ML framework, is it possible to add similar integrations with other frameworks)
* Have user impact & value that justifies the maintenance burden of supporting the feature moving forwards. The [JQuery contributor guide](https://contribute.jquery.org/open-source/#contributing-something-new) has an excellent discussion on this.

### Motivation
What is the use case in mind?  Why is it valuable to support, and why is it currently difficult or impossible to achieve? Could the desired functionality alternatively be implemented as a third-party package using MLflow public APIs? 

### Proposed Changes
For user-facing changes, what APIs are you proposing to add or modify? For internal changes, what code paths will need to be modified? 

```

Also, I wonder if we can make this more detailed to encourage users to provide more info - see the [TensorFlow RFC template](https://github.com/tensorflow/community/blob/master/rfcs/yyyymmdd-rfc-template.md) for example. In particular, IMO it'd be nice to guide users towards a workflow of:

1) Submit feature request with high-level description
2) Discuss feature request with a committer, who may ask for a more detailed design 
3) After discussion & agreement on feature request, start implementation

Maybe we should just explicitly call out ^ as the recommended workflow/lifecycle for a feature request, and encourage users to describe their designs in detail (step 2 above) using the format in the TF RFC (my suggestion above tries to incorporate some elements from the TF RFC)

Reply:
Discussed offline